packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,4 +1,73 @@
 {{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  <div class="hds-pagination-bar__total-count">1–10 of 100</div>
+
+  <nav class="hds-pagination-bar__pagination">
+    <ul class="hds-pagination-bar__page-list">
+      <li class="hds-pagination__page hds-pagination__page--previous">
+        <a class="hds-pagination__page-link" aria-label="Previous page" href="/page-?">
+          <FlightIcon @name="chevron-left" class="hds-pagination__nav-icon" /><span
+            class="hds-pagination__page-label"
+          >Previous</span>
+        </a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--current">
+        <a class="hds-pagination__page-link" href="/page-1">1</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-2">2</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-3">3</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-4">4</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-5">5</a>
+      </li>
+      <li class="hds-pagination__page">
+        <span class="hds-pagination__page-link hds-pagination__page-link--ellipsis">...</span>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-10">10</a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--next">
+        <a class="hds-pagination__page-link" href="/page-?" aria-label="Next page">
+          <span class="hds-pagination__page-label">Next</span><FlightIcon
+            @name="chevron-right"
+            class="hds-pagination__nav-icon"
+          />
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <div class="hds-pagination-bar__page-size">
+    <Hds::Form::Select::Field class="hds-pagination-bar__page-size-select" as |F|>
+      <F.Label>Items per page</F.Label>
+      <F.Options>
+        <option value="Kubernetes">10</option>
+        <option value="Other">30</option>
+        <option value="Other">50</option>
+      </F.Options>
+    </Hds::Form::Select::Field>
+  </div>
+</div>
+
+{{!
+  TEMP ROUGH NOTES
+  ------------------
+  Still figuring out class naming and DOM structure...
+
+  WIP Structure (some of these will be separated out as sub-components, others just represent HTML/CSS structure):
+    * pagination-bar
+      * total-count
+      * pagination (separate out below children or keep grouped together in one list?)
+        * page-prev
+        * pages
+          * page
+            * page-link (some content aren't links though...)
+        * page-next
+      * page-size
+}}
```

Comment:
This is still rough static DOM. Would appreciate feedback on overall structure and class-naming scheme. (Will break out into child components in subsequent steps.)

Reply:
Is it ok if I look into using Hds::Interactive as the next step? With this PR I'm focusing on the overall structure and preliminary visual styling. 

Once I start breaking things out into sub-components I can focus more on that part.